src/commands/serve/serve.mts

```diff
@@ -71,7 +72,8 @@ const serve = async (options, command) => {
   // Netlify Build are loaded.
   await getInternalFunctionsDir({ base: site.root, ensureExists: true })
 
-  let settings = /** @type {import('../../utils/types.js').ServerSettings} */ ({})
+  // @ts-expect-error TS(2571) FIXME: Type '{}' is not assignable to type 'ServerSettings'.
+  let settings: ServerSettings = {}
```

Comment:
We can get rid of the `ts-expect-error` ugliness if you change L84 to `return exit(1)` (GitHub won't let me add a suggestion for that line).

```suggestion
  let settings: ServerSettings
```

Reply:
smart! done in https://github.com/netlify/cli/pull/6183/commits/2719485ae063f5fb2813e1f3c044f5350624a501